src/Benchmarks/Middleware/JsonMiddleware.cs

```diff
@@ -43,6 +47,12 @@ public static class JsonMiddlewareExtensions
         public static IApplicationBuilder UseJson(this IApplicationBuilder builder) => builder.UseMiddleware<JsonMiddleware>();
     }
 
+    [JsonSourceGenerationOptions(PropertyNamingPolicy = JsonKnownNamingPolicy.CamelCase)]
```

Comment:
It should now be possible to do this:
```suggestion
    [JsonSourceGenerationOptions(JsonSerializerDefaults.Web)]
```